### PR TITLE
Add mod_proxy_cluster 2.x apidocs

### PR DESCRIPTION
**Pull request by jajik**
Once https://github.com/modcluster/mod_proxy_cluster/issues/158 gets merged, I'll update this if necessary and I'll mark it Ready for review.

### CODE REVIEW

File: docs/src/main/asciidoc/static/apidocs/index.html [hunk 1]
```diff
@@ -8,3 +8,12 @@ <h1>mod_cluster Javadocs</h1>
 		<a href="2.0">2.0</a>
 	</li>
 </ul>
+
+<h1>mod_proxy_cluster Doxygen</h1>
+
+<ul>
+	<li>
+		<a href="mpc-2.0">2.0</a>
```
Comment (rhusar) [inline docs/src/main/asciidoc/static/apidocs/index.html:16]: Let's add just a note here that that it's development branch, e.g. "2.0 (in development)" or something like that.

Comment (jajik) [reply docs/src/main/asciidoc/static/apidocs/index.html]: Good point, done now.

**Review by rhusar — APPROVED** [review]
LGTM

**Comment by rhusar** [comment]
Merged, publishing now, thanks @jajik

**Pull request merged**